packages/fast-development-site-react/README.md

```diff
@@ -1,2 +1,86 @@
 # FAST React development site
-This is the development site which can be used to create a react environment for a group of react or web components.
\ No newline at end of file
+This is the development site that can be used to create a React site for a library of React components. It will generate the necessary navigational elements and routing to provide a full site for the documentation and testing of components.
```

Comment:
This introductory paragraph is a little strangely worded to me. Perhaps something like:
"A boilerplate development site that can be used to show a library of React components. All necessary navigational elements and routing will be generated to provide a full site for the documentation and testing of components."

Reply:
Resolved